project/settings.py

```diff
@@ -200,13 +196,6 @@
             'mapit.context_processors.country',
             'mapit.context_processors.analytics',
         ),
-        # List of callables that know how to import templates from various sources.
-        'loaders': (
-            ('django.template.loaders.cached.Loader', (
-                'django.template.loaders.filesystem.Loader',
-                'django.template.loaders.app_directories.Loader',
-            )),
-        ),
```

Comment:
I'm sure I'm missing something simple, but how is it that this stops caching of templates in development only, and doesn't remve the cached loader entirefly?

Reply:
No, don't think you're missing anything, whoops!

Reply:
Ah, no, I wasn't! From Django 1.11, the cached loader is automatically enabled when DEBUG is False.

Reply:
Ah, OK - thanks :)